docs/4.operation/8.backlog.md

```diff
@@ -0,0 +1,41 @@
+## Backlog
```

Comment:
In general, we want to dismiss this tool by the end of FI-Next project. Therefore, I recommend that you do not include the description of it inside this repository...